sdk/eventgrid/Microsoft.Azure.Management.EventGrid/src/Microsoft.Azure.Management.EventGrid.csproj

```diff
@@ -15,7 +15,8 @@
     <TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
   </PropertyGroup>
 
-  <!-- Please do not move/edit code below this line -->
-  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.RP.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
-  <!-- Please do not move/edit code above this line -->
+  <PropertyGroup>
```

Comment:
For the management libraries I think we should keep the import. @shahabhijeet do you still generate those AzSdk.RP.props files?